index.yml

```diff
@@ -387,3 +387,13 @@ Projects:
     target-file  : Dockerfile
     notify-email : mohammed.zee1000@gmail.com
     depends-on   : null
+
+  - id          : 520
+    app-id      : gluster
+    job-id      : storagesig-gluster-centos
+    git-url     : https://github.com/gluster/docker
```

Comment:
This should probably use a repository under the @CentOS-Storage-SIG project? This version uses EPEL, that should not be needed.

Can versions for GlusterFS 3.7 and 3.8 be provided too? The repositories in the CentOS-Storage-SIG GitHub organization use branches like sig-storage7-gluster-37 and sig-storage7-gluster-38 to show the differences. In the Dockerfile, you would do `yum install centos-release-gluster37` or `yum install centos-release-gluster38`.